bottle_utils/html.py

```diff
@@ -691,6 +692,22 @@ def quoted_url(route, **params):
     return request.app.get_url(route, **quote_dict(params))
 
 
+def full_quoted_url(route, **params):
```

Comment:
Add `with_scheme` argument or something similar, because it can be useful in, say, email messages, where you need the full URL including the scheme.